source/extensions/filters/common/ext_authz/ext_authz_http_impl.cc

```diff
@@ -128,11 +128,11 @@ ClientConfig::ClientConfig(const envoy::extensions::filters::http::ext_authz::v3
       upstream_header_matchers_(toUpstreamMatchers(
           config.http_service().authorization_response().allowed_upstream_headers(),
           enable_case_sensitive_string_matcher_)),
-      authorization_headers_to_add_(
-          toHeadersAdd(config.http_service().authorization_request().headers_to_add())),
       cluster_name_(config.http_service().server_uri().cluster()), timeout_(timeout),
       path_prefix_(path_prefix),
-      tracing_name_(fmt::format("async {} egress", config.http_service().server_uri().cluster())) {}
+      tracing_name_(fmt::format("async {} egress", config.http_service().server_uri().cluster())),
+      request_headers_parser_(Router::HeaderParser::configure(
```

Comment:
should there be an overload for configure that takes the HeaderValue proto instead of converting from proto to proto?

Reply:
Yeap. I can add that.

Reply:
Done